bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/FirstCard.java

```diff
@@ -128,45 +121,34 @@ public void update() throws BankException, LoginException, BankChoiceException {
 			if (accounts.isEmpty()) {
 				throw new BankException(res.getText(R.string.no_accounts_found).toString());
 			}			
-		} catch (ClientProtocolException e) {
-			throw new BankException(e.getMessage(), e);
-		} catch (IOException e) {
-			throw new BankException(e.getMessage(), e);
-		}
-        finally {
+		} finally {
```

Comment:
We don't need a `finally` block here. If Exceptions are thrown are not `updateComplete()` either.